Caltech-IPAC/firefly
# Firefly-88: Improve coverage, catalog & Image layer handling

Firefly-88: Improve coverage/Catalog/Image layer handling 
   - icons for center and copy to clipboard
   - Coverage layer working
   - image layer will container search target and will hide image
   - new type of drawing symbol - POINT_MARKER
   - center working
   - copy to clipboard working
   - parsing CIRCE from ADQL query
   - cleaned up ToolbarButton
   - added new test to test api setting of HiPS
   - fixed api sagas that were not turned on in API Mode
   - added ACTIVE CSS style to show button press
   - coordinate sys will parse as ICRS as j2000 for now


https://jira.ipac.caltech.edu/browse/FIREFLY-88

_***To Test:***_

https://irsawebdev9.ipac.caltech.edu/firefly-88-coverage-map/firefly/

- Search a catalog, noticed improved drawing layer controls
- Gor catalog control notice how to control the color/size/symbol of the catalog and the image center separately.
- Search a second catalog, one again look a drawing layer controls
- notice base image drawing layer control
- notice new marking for the target look similar to the new icon: ![20x20-center-small](https://user-images.githubusercontent.com/6730519/61334128-65939c00-a7e6-11e9-83c0-c394167b75e5.png)

- After reload do an image search, notice drawing layers controls for image with center control enabled.
- Try new test page: https://irsawebdev9.ipac.caltech.edu/firefly-88-coverage-map/firefly/test/test-misc.html

**_Fixes:_** 

- Regression color issue in dropdown menus is fixed.
- Changing the HiPS image for coverage is not longer reverted back when a new table is loaded.
